Module/nss/pitfalltrap.nss

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please delete all nss and ncs files. All code must be in C#.

Reply:
Will clean it up.